README.md

```diff
@@ -25,6 +25,7 @@ var definition = {
     rev: '512', // optional revision number
     arch: 'amd64', // optional architecture type
     targetDir: './dist', // optional folder where to build the .deb package
+    targetName: 'my-package', // optional .deb filename
```

Comment:
already handled, package name passed as info.name option
i will document it right now